drivers/sensor/nxp/nxp_lpc_temp/temp_lpc.c

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2025 Nova Dynamics LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_lpc_temperature
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/adc.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(temp_lpc, CONFIG_SENSOR_LOG_LEVEL);
```

Comment:
Does clang-format not sort all the includes? Why is the LOG_MODULE_REGISTER needed to be between the includes?